vllm/engine/llm_engine.py

```diff
@@ -116,8 +116,13 @@ def __init__(
         # Profile the memory usage and initialize the cache.
         self._init_cache()
 
+        if self.parallel_config.sep_prompt_token:
+            # Setup the MSCCL++ communication required for KV cache transfer
+            self._setup_kvcache_comm()
```

Comment:
just curious why do we need MSCCL++ for communication, any other options like `torch.distributed`. Any analysis on the communication collective library used for KV cache transfer?